<?xml version="1.0" encoding="UTF-8" ?>
<ui version="4.0">
    <class>ConfigureCustomLayout</class>
    <widget class="QWidget" name="ConfigureCustomLayout">
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>0</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="_2">
            <item>
                <widget class="QCheckBox" name="enabled">
                    <property name="text">
                        <string>Enabled</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="screens" native="true">
                    <layout class="QVBoxLayout">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QGroupBox" name="groupBox">
                                <property name="title">
                                    <string>Top Screen</string>
                                </property>
                                <layout class="QVBoxLayout" name="verticalLayout_2">
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout">
                                            <item>
                                                <widget class="QLabel" name="label">
                                                    <property name="text">
                                                        <string>Left</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="top_left">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                                            <item>
                                                <widget class="QLabel" name="label_2">
                                                    <property name="text">
                                                        <string>Top</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="top_top">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                                            <item>
                                                <widget class="QLabel" name="label_3">
                                                    <property name="text">
                                                        <string>Right</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="top_right">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                    <property name="value">
                                                        <number>400</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                                            <item>
                                                <widget class="QLabel" name="label_4">
                                                    <property name="text">
                                                        <string>Bottom</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="top_bottom">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                    <property name="value">
                                                        <number>240</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QGroupBox" name="groupBox_2">
                                <property name="title">
                                    <string>Bottom Screen</string>
                                </property>
                                <layout class="QVBoxLayout" name="verticalLayout_3">
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_5">
                                            <item>
                                                <widget class="QLabel" name="label_5">
                                                    <property name="text">
                                                        <string>Left</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="bottom_left">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                    <property name="value">
                                                        <number>40</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_6">
                                            <item>
                                                <widget class="QLabel" name="label_6">
                                                    <property name="text">
                                                        <string>Top</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="bottom_top">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                    <property name="value">
                                                        <number>240</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_7">
                                            <item>
                                                <widget class="QLabel" name="label_7">
                                                    <property name="text">
                                                        <string>Right</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="bottom_right">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                    <property name="value">
                                                        <number>360</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_8">
                                            <item>
                                                <widget class="QLabel" name="label_8">
                                                    <property name="text">
                                                        <string>Bottom</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QSpinBox" name="bottom_bottom">
                                                    <property name="maximum">
                                                        <number>99999</number>
                                                    </property>
                                                    <property name="value">
                                                        <number>480</number>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>